### PR TITLE
Feature/dashboard

### LINKED ISSUE #191: Create Home page in Meshery UI
**Description**
Users need a landing page that both introduces them to the app and provides an overview of the status of their environment and the configuration of Meshery.

### PR DESCRIPTION
**Pull request by girishranganathan**
fixes #191

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ services:
       - "SAAS_BASE_URL=https://meshery.layer5.io"
       - "ADAPTER_URLS=meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-octarine:10003 meshery-nsm:10004"
       - "EVENT=mesheryLocal"
+    volumes:
+      - $HOME/.meshery/config:/home/appuser/.meshery/config
```
Comment (leecalcote) [inline docker-compose.yaml]: Would it be helpful for other potential use cases to mount $HOME/.meshery itself?

**Comment by leecalcote** [comment]
@girishranganathan When I build and run this branch, I'm able to navigate to the new dashboard page, but see this text on the page: "An unexpected error has occurred."

`meshery logs` produces:

```meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="Recovered from panic: runtime error: invalid memory address or nil pointer dereference"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="Recovered from panic: runtime error: invalid memory address or nil pointer dereference"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="Recovered from panic: runtime error: invalid memory address or nil pointer dereference"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"```

File: docker-compose.yaml (hunk 1 above)
Comment (girishranganathan) [reply docker-compose.yaml]: if we do that, we will also loose the yaml file

**Comment by girishranganathan** [comment]
> @girishranganathan When I build and run this branch, I'm able to navigate to the new dashboard page, but see this text on the page: "An unexpected error has occurred."
> 
> `meshery logs` produces:
> 
> ```
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="Recovered from panic: runtime error: invalid memory address or nil pointer dereference"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="Recovered from panic: runtime error: invalid memory address or nil pointer dereference"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"
> meshery_1          | time="2019-08-11T12:18:47Z" level=error msg="streaming ended with an unknown error: rpc error: code = Canceled desc = context canceled"```
> ```

Did you do a `nom I` from the UI folder?

**Pull request merged**